### PR TITLE
spire: Use short version of command docstring in output.

### LINKED ISSUE #477: Only use first docstring line as command description in spire
We get the following output from the cluster setup process:

            -- set up and verify ssh access to the supervisor node --
              --
        request SSH access to the cluster
    
        no_add_to_agent: do not add the resulting ssh key to ssh-agent
         --

This is supposed to be:

            -- set up and verify ssh access to the supervisor node --
              -- request SSH access to the cluster --

This is because the docstring on the access_ssh function looks like this:

```
@command.wrap
def access_ssh(no_add_to_agent: bool=False):
    """
    request SSH access to the cluster

    no_add_to_agent: do not add the resulting ssh key to ssh-agent
    """
```

This is a perfectly reasonable use of a docstring. But we shouldn't print the whole thing -- just the first actual line.

### PR DESCRIPTION
**Pull request by cryslith**
Fixes #477 

---

Checklist:

 - [x] I have split up this change into one or more appropriately-delineated commits.
 - [x] The first line of each commit is of the form "[component]: do something"
 - [x] I have written a complete, multi-line commit message for each commit.
 - [x] I have formatted any Go code that I have changed with gofmt.
 - ~I have written or updated appropriate documentation to cover this change.~
 - ~I have confirmed that this change is covered by at least one appropriate test run by CI.~
 - [x] If my change includes new or modified functionality, I have tested that the changes work as expected.
 - [x] I have assigned this issue to an appropriate reviewer. (Choose @celskeggs if you are not otherwise certain.)
 - [x] I consider my PR complete and ready to be merged without my further input, assuming that it passes CI and code review.
 - [x] My changes have passed CI, including an automatic Jenkins deploy.
 - [x] My changes have passed code review.

### CODE REVIEW

File: platform/spire/src/command.py [hunk 1]
```diff
@@ -43,18 +43,21 @@ def configure(self, command: list, parser: argparse.ArgumentParser):
         subparsers = parser.add_subparsers()
 
         for component, subcommand in self.mapping.items():
-            doc = inspect.getdoc(subcommand)
+            doc = subcommand.short_doc()
             short_doc = doc.split('\n')[0] if doc else None
```
Comment (celskeggs) [inline platform/spire/src/command.py]: Are these local variables used for anything anymore?

File: platform/spire/src/command.py [hunk 2]
```diff
@@ -68,6 +71,9 @@ def configure(self, command: list, parser: argparse.ArgumentParser):
         add_dry_run_argument(parser, 'dry_run_outer')
         add_show_commands_argument(parser, 'show_commands_outer')
 
+    def short_doc(self):
+        return self.__doc__
+
```
Comment (celskeggs) [inline platform/spire/src/command.py]: Why do we have to override the superclass's implementation? Isn't this just the same implementation as Mux?

File: platform/spire/src/command.py (hunk 1 above)
Comment (cryslith) [reply platform/spire/src/command.py]: oops

File: platform/spire/src/command.py (hunk 2 above)
Comment (cryslith) [reply platform/spire/src/command.py]: oops

File: platform/spire/src/command.py (hunk 1 above)
Comment (cryslith) [reply platform/spire/src/command.py]: on the bright side - pyflakes caught this in CI :)

**Pull request merged**